src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountVerificationService.java

```diff
@@ -54,19 +72,40 @@ private void sendMediaUsersForVerification() {
                     azureUserService.getUser(user.getEmail()).givenName
                 ));
             } catch (AzureCustomException ex) {
-                log.error("Error when getting user from azure: %s", ex.getMessage());
+                log.error(writeLog("Error when getting user from azure: " + ex.getMessage()));
             }
         });
     }
 
     /**
-     * Method that gets all media users who have not verified their account in 365 days.
+     * Method that gets all AAD amin users who last signed in at least 76 days (by default) ago, and gets all
+     * IDAM users who last signed in at least 118 days (by default) ago.
+     * Then send their details on to publication services to send them a notification email.
+     */
+    private void notifyAadAdminAndIdamUsersToSignIn() {
+        // TODO: this is a placeholder. Needs to be updated with call to Publication services for notifying the users.
+        Stream.of(
+                userRepository.findAadAdminUsersByLastSignedInDate(aadAdminAccountSignInNotificationDays),
+                userRepository.findIdamUsersByLastSignedInDate(idamAccountSignInNotificationDays))
+            .flatMap(Collection::stream)
+            .forEach(user -> log.info(writeLog("Remind user to sign in: " + user.getEmail())));
```

Comment:
We should either mask the email or replace the users email with an ID just so it's not logged out